app/main.py

```diff
@@ -16,4 +18,21 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+
+sorted_variables = {"mutable": [], "immutable": []}
+
+def sort_variables(*args) -> None:
+
+    for current_variable in args:
+        what_type = str(type(current_variable))
+        key = "immutable"
+
+        if "dict" in what_type or "list" in what_type or "set" in what_type:
+            key = "mutable"
+
+        sorted_variables[key].append(current_variable)
+
+
+sort_variables(lucky_number, pi, one_is_a_prime_number)
+sort_variables(name, my_favourite_films, profile_info)
+sort_variables(marks, collection_of_coins)
```

Comment:
It's not clear why you are calling the `sort_variables()` function multiple times. It would be more efficient to call this function once with all the variables you want to sort.